### PR TITLE
Email for a new private bugnote was send to a non authorized reporter

### PR DESCRIPTION
**Pull request by wuttke**
see https://www.mantisbt.org/bugs/view.php?id=22898

### CODE REVIEW

File: core/bug_api.php [hunk 1]
```diff
@@ -1989,10 +1989,13 @@ function bug_reopen( $p_bug_id, $p_bugnote_text = '', $p_time_tracking = '0:00',
  * @access public
  * @uses database_api.php
  */
-function bug_update_date( $p_bug_id ) {
+function bug_update_date( $p_bug_id, $p_last_modified = 0 ) {
```
Comment (dregad) [inline core/bug_api.php]: Since you change the function's signature, you need to update the PHPDoc also.

**Comment by dregad** [comment]
Also, please reword the commit message to explain what you changed and why, and adjust the issue reference (`Fixes #22898`) otherwise it won't be picked up by source integration.
Thanks

**Comment by vboctor** [comment]
I have added a high level question to the bug.

**Comment by vboctor** [comment]
The associated issue was closed with a commit from @dregad - closing PR.

**Pull request closed without merging**